### PR TITLE
Issue #940: Set default path patterns for all core node types.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/940

### CODE REVIEW

**Comment by jenlampton** [comment]
omg regular expressions...  @quicksketch I'm going to need your regex-fu for help on this one.  Here is  part of a regular expression:
```
$outline .= '(node\/' . $node->nid . ')(.*?)(' . $node->title . ')(.*?)';
```
We need to replace the `node/NID` in this with the `url()`ed equivalent path.

File: core/modules/path/tests/path.test [hunk 1]
```diff
@@ -72,6 +75,7 @@ class PathTestCase extends BackdropWebTestCase {
     // Change alias to one containing "exotic" characters.
     $pid = $this->getPID($edit['alias']);
 
+
```
Comment (docwilmot) [inline core/modules/path/tests/path.test]: Blank line.

Comment (jenlampton) [inline core/modules/path/tests/path.test]: hah

Comment (jenlampton) [reply core/modules/path/tests/path.test]: pushed a fix for that :)

**Comment by quicksketch** [comment]
Merged into 1.x for 1.5.0. Thanks!

**Pull request merged**